pfioh/swift_store.py

```diff
@@ -72,21 +87,40 @@ def storeData(self, **kwargs):
                     break
                 f.write(chunk)
             f.close()
+
+        zip_file_contents = open('/tmp/{}.zip'.format(key), mode='rb')
+
         try:
             success = True
-            uploadObject = swift_service.SwiftUploadObject('/tmp/{}.zip'.format(key), object_name="input/data")
-            uploadResultsGenerator = swiftService.upload(str_containerName, [uploadObject])
-            # generates dicts containing the results of the upload
-            for res in uploadResultsGenerator:
-                print("Upload results generated")
-                if not res["success"]:
-                    success = False
-                pp.pprint(res)
+            filePath = "input/data"
+
+            resp_headers, containers = swiftService.get_account()
+            listContainers = [d['name'] for d in containers if 'name' in d]
```

Comment:
Why are we listing the switft containers? Just to be sure that this container is present, if not create it?

Reply:
The containers are being listed to ensure that a new container is not being created if another container with the same name exists.